src/components/Contact/Contact.js

```diff
@@ -1,23 +1,24 @@
-import React, { Component } from 'react';
-// import '../../styles/Contact.css';
+import React, { Component } from "react";
+import "../../styles/Contact.css";
 
-// import Header from './Header.js';
-// import Chat from './Chat.js';
-// import Help from './Help.js';
-// import SocialMediaIcons from './SocialMediaIcons.js'
-// import Follow from './Follow.js'
-// import Copyright from './Copyright.js'
+// Components
+import Header from "./Header.js";
+import Chat from "./Chat.js";
+import Help from "./Help.js";
+import SocialMediaIcons from "./SocialMediaIcons.js";
+import Follow from "./Follow.js";
+import Copyright from "./Copyright.js";
```

Comment:
Contact.js: Code is clean and well-organized for future additions/amendments. Any particular reason for the change from `import React, { Component } from 'react';` to `import React, { Component } from "react";`? Any particular reason for the change from `import Header from './Header.js';` etc. to `import Header from "./Header.js";` etc.?